sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -185,6 +189,22 @@ def close(self):
     def sleep(self, duration):  # pylint: disable=no-self-use
         time.sleep(duration)
 
+    @property
```

Comment:
These are the properties I've added to transport, to be able to support multiple formats